src/commands/NodeActionCommands.tsx

```diff
@@ -910,4 +910,93 @@ export function addNodeActionCommands(
         
         }
     }
+
+    // Add command to attach selected node
+    commands.addCommand(commandIDs.attachNode, {
+        execute: async () => {
+
+            const widget = tracker.currentWidget?.content as XircuitsPanel;
+            const model = widget.xircuitsApp.getDiagramEngine().getModel();
+            const selected_entities = model.getSelectedEntities();
+            const selected_nodes = selected_entities.filter(entity => entity instanceof NodeModel) as CustomNodeModel[];
+            selected_nodes.forEach(node => {
+                node.getOptions().selected = false
```

Comment:
There is a `setSelected(val)` node, we might as well use it. I know it does the same thing under the hood, but still `node.setSelected(false)` reads better.

Reply:
Ah good one. Updated. Maybe we'd want a set attached method in the future.